internal/mode/static/state/graph/policies_test.go

```diff
@@ -192,30 +115,63 @@ func TestAttachPolicies(t *testing.T) {
 	}
 
 	tests := []struct {
```

Comment:
I did try this but was seeing failure on multiple runs, just tried as well too. I don't get the stack trace though. 
Sadly, didn't get a stack trace but pretty sure its related to this because these tests passed yesterday when I ran it 10-12 times. 

```
There were failures detected in the following suites:
  static ./internal/mode/static
```

I feel this is also a cleaner way to write this unit test since inputs are visible when looking at the test struct. Will keep the change.